src/site/content/en/handbook/content-checklist/index.md

```diff
@@ -10,9 +10,9 @@ description: |
 This is a checklist for web.dev content reviews. It includes critical high-level issues and lower-level issues that commonly appear in drafts.
 
 ## Basics
-1. Is the piece providing value for one or more of the web.dev [personas](/handbook/audience)?
-1. Do the title, subtitle, and description convey the value proposition?
-1. Is the piece 1,000 words or less—or is there a good reason for it to be longer?
+1. Does the piece provide value for one or more of the web.dev [personas](/handbook/audience)?
+1. Do the title, subtitle, and description convey a [value proposition](/handbook/audience/#value-propositions)?
+1. Is the piece 1,000 words or less? If it's longer, is there a good reason for it to be longer?
```

Comment:
what's a good reason?  Is there a decision matrix for exceptions?

Reply:
I mean, no, haha. I was hoping to keep this more ad hoc?

Reply:
Actually there are good reasons. For some new, large APIs, 1000 words is not sufficient to capture what needs to be said. WebXR Device API, for which I'll be submitting articles in September suffers from this. I plan to address this by proposing a multi-part series. Do we have guidance for such a creature?